tests/reg_tests/test_DVConstraints.py

```diff
@@ -810,6 +812,137 @@ def test_14(self, train=False, refDeriv=False):
+            handler.root_add_dict("funcs_base", funcs, rtol=1e-6, atol=1e-6)
+
+            handler.assert_allclose(
+                funcs["DVCon1_trivolume_constraint_1"], 1.0, name="scaled_volume_base", rtol=1e-7, atol=1e-7
+            )
+
+            # BWB volume computed with meshmixer
+            handler.assert_allclose(
+                funcs["unscaled_vol_con"], 1103.57, name="unscaled_volume_base", rtol=1e-7, atol=1e-7
+            )
+
+            funcsSens = {}
+            DVCon.evalFunctionsSens(funcsSens, includeLinear=True)
+            # regress the derivatives
+            handler.root_add_dict("derivs_base", funcsSens, rtol=1e-6, atol=1e-6)
+            # FD check DVGeo1
+
+            funcsSensFD = evalFunctionsSensFD(DVGeo1, DVCon, fdstep=1e-6)
+            at_least_one_var = False
+            for outkey in funcs.keys():
+                for inkey in DVGeo1.getValues().keys():
+                    analytic = funcsSens[outkey][inkey]
+                    fd = funcsSensFD[outkey][inkey]
+                    handler.assert_allclose(analytic, fd, name="finite_diff_check", rtol=1e-5, atol=1e-5)
+                    # make sure there are actually checks happening
+                    self.assertTrue(np.sum(np.abs(fd)) > 1e-10)
+                    at_least_one_var = True
+            self.assertTrue(at_least_one_var)
+
+
+class RegTestGeograd(unittest.TestCase):
+
+    N_PROCS = 1
+
+    def setUp(self):
+        # Store the path where this current script lives
+        # This all paths in the script are relative to this path
+        # This is needed to support testflo running directories and files as inputs
+        self.base_path = os.path.dirname(os.path.abspath(__file__))
+
+    def bwb_test_twist(self, DVGeo, DVCon, handler):
+        funcs = {}
+        funcsSens = {}
+        # change the DVs
+        xDV = DVGeo.getValues()
+        xDV["twist"] = np.linspace(0, 10, self.nTwist)
+        DVGeo.setDesignVars(xDV)
+        # check the constraint values changed
+        DVCon.evalFunctions(funcs, includeLinear=True)
+
+        handler.root_add_dict("funcs_twisted", funcs, rtol=1e-6, atol=1e-6)
+        # check the derivatives are still right
+        DVCon.evalFunctionsSens(funcsSens, includeLinear=True)
+        # regress the derivatives
+        handler.root_add_dict("derivs_twisted", funcsSens, rtol=1e-6, atol=1e-6)
+        return funcs, funcsSens
+
+    def bwb_test_deformed(self, DVGeo, DVCon, handler):
+        funcs = {}
+        funcsSens = {}
+        xDV = DVGeo.getValues()
+        np.random.seed(37)
+        xDV["local"] = np.random.normal(0.0, 0.05, 32)
+        DVGeo.setDesignVars(xDV)
+        DVCon.evalFunctions(funcs, includeLinear=True)
+        DVCon.evalFunctionsSens(funcsSens, includeLinear=True)
+        handler.root_add_dict("funcs_deformed", funcs, rtol=1e-6, atol=1e-6)
+        handler.root_add_dict("derivs_deformed", funcsSens, rtol=1e-6, atol=1e-6)
+        return funcs, funcsSens
+
```

Comment:
Is there a way to directly skip the class initialization down here? no biggie though

Reply:
Despite being outdated, I still refer to the `RegTestGeograd` class

Reply:
You mean skip the class instead of each test if geograd is not installed? Yes, I think that's possible. I will change that.

Reply:
I don't think we do it in other repos, but if you find a way it could be nice. The initialization time for pyGeo is not that bad, but the solvers might benefit from this?